xsd/netex_framework/netex_reusableComponents/netex_vehicleType_version.xsd

```diff
@@ -479,6 +479,11 @@ Rail transport, Roads and Road transport
 					<xsd:documentation>Category of vehicle.</xsd:documentation>
 				</xsd:annotation>
 			</xsd:element>
+			<xsd:element name="NumberOfWheels" type="xsd:nonNegativeInteger" minOccurs="0">
```

Comment:
I am looking forward to the first levitating vehicle _without_ wheels.

Reply:
![image](https://github.com/user-attachments/assets/8da096ad-4e48-426b-82da-881f400b361b)